tools/llm_bench/llm_bench_utils/model_utils.py

```diff
@@ -181,18 +181,26 @@ def analyze_args(args):
             model_args['config'] = config
     if model_framework == 'ov':
         set_default_param_for_ov_config(model_args['config'])
+        if 'ATTENTION_BACKEND' not in model_args['config'] and use_case in ['text_gen', 'vlm'] and args.device != "NPU":
+            model_args['config']['ATTENTION_BACKEND'] = PA_ATTENTION_BACKEND
+        if model_args['config'].get('ATTENTION_BACKEND', '') == PA_ATTENTION_BACKEND and args.device == "NPU":
+            model_args['config']['ATTENTION_BACKEND'] = SDPA_ATTENTION_BACKEND
+            log.warning("Continuous Batching, Speculative decoding and Prompt Lookup decoding is not supported for NPU device")
         log.info(f"OV Config={model_args['config']}")
     elif model_framework == 'pt':
         log.info(f"PT Config={model_args['config']}")
     model_args['model_type'] = get_model_type(model_name, use_case, model_framework)
     model_args['model_name'] = model_name
 
-    if use_cb and optimum:
+    if model_args['config'].get('ATTENTION_BACKEND', '') == PA_ATTENTION_BACKEND and optimum:
```

Comment:
probably now you need an additional check that if SDPA in attention backed and speculative decoding selected